Formula/tbb.rb

```diff
@@ -23,9 +23,13 @@ class Tbb < Formula
   def install
     compiler = (ENV.compiler == :clang) ? "clang" : "gcc"
     args = %W[tbb_build_prefix=BUILDPREFIX compiler=#{compiler}]
-
     system "make", *args
     lib.install Dir["build/BUILDPREFIX_release/*.dylib"]
+
+    args = %W[tbb_build_prefix=BUILDPREFIX compiler=#{compiler} extra_inc=big_iron.inc]
+    system "make", *args
+    lib.install Dir["build/BUILDPREFIX_release/*.a"]
+    
```

Comment:
Style audit failure for trailing whitespace:

```
Error Message
failed: brew style tbb
Stacktrace
        warning: parser/current is loading parser/ruby23, which recognizes
warning: 2.3.7-compliant syntax, but you are running 2.3.3.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
== /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/tbb.rb ==
C: 32:  1: Trailing whitespace detected.

1 file inspected, 1 offense detected
```

Could you check for trailing whitespace here?